### ISSUE TITLE
Add Cryto Algos

### ISSUE THREAD

**Issue by Ankit1598** [Ankit1598/crypto-algos#11]
Make PR's for the algorithms related to Crytography.

Make sure you add the algorithm inside a folder with the algorithm name. If the folder exists add you code in it and if the algorithm is not yet been added in the repo make a new folder accordingly.

Thank You

**Comment by romasinha** [comment]
Hey @Ankit1598 can I make a contribution to this?

**Comment by Ankit1598** [comment]
> Hey @Ankit1598 can I make a contribution to this?

yes ofcourse this is a open repo and u can make as many contributions u want 👍
